### PR TITLE
Java Wrapper | Fix SAST & IAC Vul + ThresholdAST-47965)

### PR DESCRIPTION
**Pull request by Korjen97**

### CODE REVIEW

**Comment by AlvoBen** [comment]
Logo
**Checkmarx One – Scan Summary & Details** – [cdbeb009\-7d17\-48f3\-9e74\-aa0dbd1a2a35](https://eu.ast.checkmarx.net/projects/df2dd522-8671-4034-acc7-e9fdd567fb26/scans?id=cdbeb009-7d17-48f3-9e74-aa0dbd1a2a35&branch=itay%2FFix-sast-iac-vul)
### Fixed Issues
Severity|Issue|Source File / Package
---|---|---
HIGH|Passwords And Secrets \- Generic Password|/ci\.yml: [40](https://github.com/CheckmarxDev/ast-cli-java-wrapper/blob/itay/Fix-sast-iac-vul//.github/workflows/ci.yml#L40)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ jobs:
           java-version: '11'
           server-id: ossrh
           server-username: MAVEN_USERNAME
-          server-password: MAVEN_PASSWORD
+          server-password: ${{ secrets.OSSRH_TOKEN }}
```
Comment (tamarleviCm) [inline .github/workflows/ci.yml:40]: Do we have this secret?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -76,6 +76,6 @@ jobs:
 
       - name: Run SpotBugs Analysis
         if: ${{ github.actor != 'dependabot[bot]' }}
-        uses: jwgmeligmeyling/spotbugs-github-action@master
+        uses: jwgmeligmeyling/spotbugs-github-action@b8e2c3523acb34c87f14e18cbcd2d87db8c8584e #master
```
Comment (tamarleviCm) [inline .github/workflows/ci.yml:79]: instead of master put the current tag version

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -74,7 +74,7 @@ jobs:
           distribution: 'temurin'
           server-id: ossrh
           server-username: MAVEN_USERNAME
-          server-password: MAVEN_PASSWORD
+          server-password: ${{ secrets.OSSRH_TOKEN }}
```
Comment (tamarleviCm) [inline .github/workflows/release.yml:77]: again - do we have this secret?

**Review by tamarleviCm — APPROVED** [review]

**Pull request merged**